doc-content/enterprise-only/installation/clustering-bc-configure-standalone_proc.adoc

```diff
@@ -0,0 +1,65 @@
+====
+
+.Prerequisites
+
+* Elasticsearch is installed as described in <<clustering-elasticsearch-proc_{context}>>.
+* {EAP} is installed on each node of the cluster as described in <<clustering-download-eap-bc-proc>>.
+* An NFS server with a mounted partition is available.
+
+.Procedure
+. Open the `_EAP_HOME_/standalone/configuration/standalone-full-ha.xml` file in a text editor.
+. Edit or add the following properties to the `<system-properties>` element and replace the following placeholders:
+* `<ELASTICSEARCH_NODE_IP>` is the IP addresses where Elasticseach is installed.
+* `<JMS_USER>` is a {EAP} application user. It is the user that {CENTRAL} uses to connect to the  embedded JMS broker. 
+* `<JMS_USER_PASSWORD>` is the password for `<JMS_USER>`.
++
+[source,xml]
+----
+<system-properties>
+  <property name="appformer-jms-connection-mode" value="REMOTE"/>
+  <property name="appformer-jms-username" value="<AMQ_USER>  "/>
+  <property name="appformer-jms-password" value="<AMQ_USER_PASSWORD>"/>
+  <property name="appformer-jms-url" value="tcp://<AMQ_BROKER_IP_ADDRESS>:61616?ha=true&retryInterval=1000&retryIntervalMultiplier=1.0&reconnectAttempts=-1"/>
+  <property name="org.appformer.ext.metadata.elastic.port" value="9300"/>
+  <property name="org.appformer.ext.metadata.elastic.host" 
+      value="<ELASTICSEARCH_NODE_IP>"/>
+  <property name="org.appformer.ext.metadata.elastic.cluster" 
+      value="kie-cluster"/>
+  <property name="org.appformer.ext.metadata.index" value="elastic"/> 
+  <property name="org.appformer.ext.metadata.elastic.retries" value="10"/>
+  <property name="appformer-jms-connection-factory" value="java:/ConnectionFactory"/> 
+  <property name="org.uberfire.nio.git.dir" value="<SHARED_NETWORK_FOLDER>"/>
+</system-properties>
+----
+.  In the `<subsystem xmlns="urn:jboss:domain:messaging-activemq:2.0">` element, make the following changes:
+.. Replace `<cluster password="${jboss.messaging.cluster.password:CHANGE ME!!}"/>` with `<cluster user="<CLUSTER_USER>" password="<CLUSTER_PWD>"/>` where `<CLUSTER_USER>` is the user used by {EAP} to form a messaging cluster and `<CLUSTER_PWD>` is the password for that user.
+.. In the `security-setting` element , add the role of the `<JMS_USER>` to a new `role-name` element, for example:
++
+[source]
+----
+<security-setting name="#">
+    <role name="guest" send="true" consume="true" create-non-durable-queue="true" delete-non-durable-queue="true"/>
+    <role name="admin" send="true" consume="true" create-non-durable-queue="true" delete-non-durable-queue="true"/>
+</security-setting>
+----
+. Save the `standalone-full-ha.xml` file.
+. To start the cluster, navigate to `__EAP_HOME__/bin` and enter one of the following commands:
+** On Linux or UNIX-based systems:
++
+[source,bash]
+----
+$ ./standalone.sh -c standalone-full.xml
```

Comment:
The steps above talk about editing standalone-full-ha.xml, but here you're using standalone-full.xml.
 Lines 58 and 64 should mave standalone-full **-ha** .xml